### PR TITLE
Getting values from configuration.json file

### PR DESCRIPTION
**Pull request by shmuelko**
configuration.json file will fill the fields (or replace the values from the user input) URI, email, project-id, API-token, and additional-test-fields.
Additionally, the values form the XML will be taken from all the available fields.

### CODE REVIEW

File: src/practitest_firecracker/cli.clj [hunk 1]
```diff
@@ -23,6 +23,14 @@
                "Directory doesn't exist"]
     :assoc-fn (fn [m k v]
                 (reduce #(update %1 k conj %2) m v))]
+   [nil "--config-path PATH"
+    "Path to surefire cofiguration file"
```
Comment (stask) [inline src/practitest_firecracker/cli.clj]: Just `Path to configuration file` or `Path to firecracker configuration file`.

File: src/practitest_firecracker/cli.clj [hunk 2]
```diff
@@ -23,6 +23,14 @@
...
+    :parse-fn #(string/split % #"\s*,\s*")
```
Comment (stask) [inline src/practitest_firecracker/cli.clj]: This is only needed when the parameter is specified more than once (for reports paths for example). Configuration file can be specified only once.

File: src/practitest_firecracker/cli.clj [hunk 3]
```diff
@@ -77,52 +85,66 @@
   (format "%s is required for '%s' action" option-name action-name))
 
 (defn parse-args [args]
-  (let [{:keys [options arguments errors summary]} (parse-opts args cli-options)]
+  (let [{:keys [options arguments errors summary]} (parse-opts args cli-options)
+        config-content (with-open [rdr (reader (str (first (:config-path options))))]
+                         (reduce conj [] (line-seq rdr)))
+        parsed-json    (json/parse-string (first config-content) true)
```
Comment (stask) [inline src/practitest_firecracker/cli.clj]: you can do it in one line: `(json/parse-stream (reader <PATH>))`

File: src/practitest_firecracker/cli.clj [hunk 4]
```diff
@@ -77,52 +85,66 @@
...
+        new-additional-test-fields  (parse-additional-fields (json/generate-string (:additional-test-fields parsed-json)))
+        new-additional-testset-fields (parse-additional-fields (json/generate-string (:additional-testset-fields parsed-json)))
+        new-parsed-json    (merge parsed-json {:additional-testset-fields new-additional-testset-fields
+                                               :additional-test-fields new-additional-test-fields
+                                               })
+        options2 (merge options new-parsed-json)]
```
Comment (stask) [inline src/practitest_firecracker/cli.clj]: Why do you need `options2`? Just use `options (merge options new-parsed-json)`.

**Review by stask — CHANGES_REQUESTED** [review]

File: src/practitest_firecracker/cli.clj (hunk 3 above)
Comment (shmuelko) [reply src/practitest_firecracker/cli.clj]: you mean like this?: (json/parse-stream (reader (:config-path options)))

File: src/practitest_firecracker/cli.clj (hunk 4 above)
Comment (shmuelko) [reply src/practitest_firecracker/cli.clj]: that was for debugging

File: src/practitest_firecracker/cli.clj (hunk 1 above)
Comment (shmuelko) [reply src/practitest_firecracker/cli.clj]: Will fix

File: src/practitest_firecracker/cli.clj (hunk 2 above)
Comment (shmuelko) [reply src/practitest_firecracker/cli.clj]: Will remove ":prase-fn"

**Review by stask — APPROVED** [review]

**Pull request merged**